cvat-core/src/api.js

```diff
@@ -470,34 +469,42 @@ function build() {
                 return result;
             },
             /**
-                * Install plugin to CVAT
-                * @method register
-                * @async
-                * @memberof module:API.cvat.plugins
-                * @param {Plugin} [plugin] plugin for registration
-                * @throws {module:API.cvat.exceptions.PluginError}
-            */
+             * Install plugin to CVAT
+             * @method register
+             * @async
+             * @memberof module:API.cvat.plugins
+             * @param {Plugin} [plugin] plugin for registration
+             * @throws {module:API.cvat.exceptions.PluginError}
+             */
             async register(plugin) {
                 const result = await PluginRegistry
                     .apiWrapper(cvat.plugins.register, plugin);
                 return result;
             },
         },
+
+        allowedApps: {
+            async list() {
+                const result = await PluginRegistry.apiWrapper(cvat.allowedApps.list);
+                return result;
+            },
+        },
```

Comment:
Why not to use ``plugins.list()`` to get this information?
``plugin.list()`` is used to get available apps from the server ``analytics``, ``git``.

Reply:
Because the same elements shoud have the same behavior. But now analytics, models and tasks implementation is different. And implementation of analitycs and git different also. It first step to start working with them in the same way.

Reply:
What do you mean speaking that implementation is different?
Right now ``models`` are built-in like ``tasks``. 

Checking whether server has applications ``git`` and ``analytics`` happens from ``cvat-ui`` (in ``plugin-cheker.ts``) what is of course wrong approach (need to test it via ``cvat-core`` [where we might add method like ``serverList`` to namespace ``plugins``]), but in context of cvat client, ``analytics`` and ``git`` are plugins.

The patch introduced some rather abstract ``meta`` what is actually looks like the same what is plugins.

Reply:
And speaking about the PR's name. How is it related with these code changes?
I just try to understand what task you are trying to solve

Reply:
The goal of this pr hide the models button